packages/experimental/Divider/src/Divider.styling.ts

```diff
@@ -0,0 +1,149 @@
+import { useMemo } from 'react';
+import { ViewProps } from 'react-native';
+import { Theme } from '@fluentui-react-native/framework';
+import { IconPropsV1 as IconProps } from '@fluentui-react-native/icon';
+import { TextProps } from '@fluentui-react-native/text';
+import { DividerTokens, DividerAppearance } from './Divider.types';
+
+const getIconProps = (tokens: DividerTokens, icon: IconProps): IconProps => {
+  if (icon.fontSource) {
+    return {
+      fontSource: {
+        ...icon.fontSource,
+        color: tokens.color || tokens.contentColor,
```

Comment:
Which token takes precedence? And why are there 2 very similar tokens? Can we just use one?

Reply:
The color token is directly set by the user through a prop, whereas there are two other tokens, contentColor and lineColor, that are indirectly set by the appearance prop. 

By default, the lines and content in the divider are shaded slightly differently, but if a user sets a color, then that color takes precedence over appearance.

Reply:
If the color token is directly set by a prop, should it just be a prop? Is there any value keeping it a token? I know we have the whole `tokens that are also props` thing but I found that to be somewhat confusing / an escape hatch 😅